{
  try {
    _persistence.countByR_N(RandomTestUtil.nextLong(),RandomTestUtil.nextLong());
    _persistence.countByR_N(0L,0L);
  }
 catch (  Exception e) {
    Assert.fail(e.getMessage());
  }
}
